openmrs-sdk-maven-plugin-integration-test/pom.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.openmrs.maven.plugins</groupId>
+        <artifactId>openmrs-sdk-maven-plugin-main</artifactId>
+        <version>2.1.3-SNAPSHOT</version>
```

Comment:
Let's make the parent org.openmrs.maven:openmrs-sdk:2.1.3-SNAPSHOT. The reasoning is that we will want to include different artifacts (not only plugins, but also archetypes) within the sdk.